documentation/src/main/asciidoc/user_guide/server_protocols_hotrod.adoc

```diff
@@ -1062,10 +1066,38 @@ marshaller class implementation.
 3. Deploy the JAR file in the {brandname} Server.
 
 Note that the Marshaller could be deployed in either a separate jar, or in the
-same jar as the `CacheEventConverter` and/or `CacheEventFilter` instances. Also, currently
-deployment of a single `org.infinispan.commons.marshall.Marshaller` instance
-is supported. If multiple marshaller instances are deployed, warning messages
-will be displayed as reminder indicating which marshaller instance will be used.
+same jar as the `CacheEventConverter` and/or `CacheEventFilter` instances.
+
+===== Protostream deployment
```

Comment:
```Protobuf deployment``` would sound better. No need to stress on protostream, it's just an implementation detail. Nobody knows what protostream is :)

Reply:
I think I meant "Protostream marshallers deployment"